utils/build.ps1

```diff
@@ -629,12 +724,30 @@ function Build-CMakeProject {
     # Add additional defines (unless already present)
     $Defines = $Defines.Clone()
 
+    # TODO(compnerd) clean up these and formalise this selection
+    if ($Platform -eq "Android") {
+      $env:Path = "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin;C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja;${env:Path}"
```

Comment:
Did the trick I suggested with `Isolate-EnvVars` and `Invoke-VsDevShell` not work?

Reply:
I need to try to implement them. Diff welcome of course :)